de/ostern/2017/index.html

```diff
@@ -6,21 +6,27 @@
 <meta name="keywords" content="Ostereier, Ostern, Harry, Potter, HP-FC">
 <meta name="description" content="In der Ostereiersuche des HP-FCs hat der Osterhase viele Ostereier versteckt. Finde die Oster-Eier in den selbsgemalten Bildern um zu sehen, was sich hinter ihnen verbirgt! Es gibt Geschichten, Gedichte, Bastelanleitungen und Backrezepte zu finden. Findest du alle Eier?">
 <script src="../../../js/ostern/bower_components/webcomponentsjs/webcomponents-lite.js"></script>
+<link rel="import" href="../../../js/ostern/bower_components/polymer/polymer.html">
+<link rel="import" href="../../../js/ostern/bower_components/polymer/lib/elements/dom-if.html">
 <link rel="import" href="../../../js/ostern/oster-projekt.html">
+<link rel="import" href="../../../js/ostern/oster-bild.html">
+<link rel="import" href="../../../js/ostern/oster-text.html">
+<link rel="import" href="../../../js/ostern/oster-ei.html">
 </head>
 
-<body unresolved>
+<body>
 	<!--[if lt IE 10]><div style="background-color: #ebebeb;">Dein Browser unterstützt die Ostereiersuche leider nicht. Bitte verwende einen modernen Browser.</div><![endif]--><!--[if gte IE 10]> -->
-	
+	<oster-hpfc-menu></oster-hpfc-menu>
 	<oster-projekt
 		headerlogo="../../../img/ostern/2017/logo.gif"
 		impressumhref="impressum.html"
-		cookiename="ostern2017">
+		cookiename="ostern2018">
 		<h1>HP-FC – Ostereiersuche 2017</h1>
-
-		<oster-bild src="../../../img/ostern/2017/rav-suchbild.jpg">
+		<oster-bild slot="oster-bild" src="../../../img/ostern/2017/rav-suchbild.jpg">
```

Comment:
Hm, die Wiederholung des Element-Namens in `slot="oster-bild"` ist alles andere als DRY 😞

Kannst du die jeweils per JavaScript bei der Element-Konstruktion hinzufügen? Ist ja ein fixer Wert pro Element und müsste damit prima klappen. Damit haben wir das aus dem HTML raus.